src/renderGraphiQL.js

```diff
@@ -72,6 +82,7 @@ add "&raw" to the end of the URL within a browser.
   <script src="//cdn.jsdelivr.net/react/15.4.2/react.min.js"></script>
   <script src="//cdn.jsdelivr.net/react/15.4.2/react-dom.min.js"></script>
   <script src="//cdn.jsdelivr.net/npm/graphiql@${GRAPHIQL_VERSION}/graphiql.min.js"></script>
```

Comment:
ideally we should manage the react and other client-side libs via npm or bower, npm is recommended, then browserify and create bundle,  in that way, some issue like #112 can be resolved.